care/facility/api/serializers/patient.py

```diff
@@ -518,6 +532,17 @@ class PatientNotesSerializer(serializers.ModelSerializer):
     thread = serializers.ChoiceField(
         choices=PatientNoteThreadChoices, required=False, allow_null=False
     )
+    reply_to = ExternalIdSerializerField(
+        queryset=PatientNotes.objects.all(),
+        required=False,
+        allow_null=True,
+    )
+
+    def to_representation(self, instance):
+        ret = super().to_representation(instance)
+        if instance.reply_to:
+            ret["reply_to"] = ReplyToPatientNoteSerializer(instance.reply_to).data
+        return ret
```

Comment:
We could simply do this instead right?


```suggestion
    reply_to_object = ReplyToPatientNoteSerializer(source="reply_to", read_only=True)
```